pom.xml

```diff
@@ -128,13 +128,25 @@
 			<artifactId>cloudfoundry-client-reactor</artifactId>
 			<version>${cloudfoundry.client.version}</version>
 		</dependency>
+		<!--Need to override version of compress due to CVE-2019-12402 -->
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-compress</artifactId>
+            <version>1.19</version>
+		</dependency>
```

Comment:
I am in general not in favor of changing versions of transitive dependency - often, it's asking for trouble. The only exemption to the rule is security (which you have pulled here 😁 )

The dependency is coming from cloudfoundry/cf-java-client (here: cloudfoundry-client-reactor:3.15.0.RELEASE`). The version we are currently using has this dependency to `commons-compress:1.18`.
cloudfoundry/cf-java-client has released [3.17.1.RELEASE in the meantime](https://github.com/cloudfoundry/cf-java-client/releases/tag/v3.17.1.RELEASE). That uses `commons-compress:1.19` out of the box. So, we wouldn't have to provide a manual overwrite of the version. Hence, I am currently more in favor of bumping the version of the entire artifact. What do you think?

Reply:
I would bump the entire artifact , that is cleanest. My only concern was any subtle behaviour changes between cf client versions, which is why I chose to override as the safest solution.

Reply:
> My only concern was any subtle behaviour changes between cf client versions

I do understand your concern, which I share with you in general. However, in this special case, there are two reasons for me why I think it's not a problem:
1. Compatibility of cloudfoundry/cf-java-client (except for bugs 😀) is really good so far.
2. We are talking here about `master`. We cannot dig ourselves in to a single version: Not bumping now would mean that we have to fear bumping also in the feature. That's not acceptable.

I have added efe9e52caa9516f942b237480684cdf43f6ca48a which bumps the surrounding library version.